src/List.js

```diff
@@ -66,13 +86,33 @@ export function List() {
 
   const handleChange = async (id, event) => {
     let date = new Date();
+    const item = items.find((element) => element.id === id);
+    const daysSinceLastTransaction = item.lastPurchasedDate
+      ? Math.round((new Date() - item.lastPurchasedDate) / 1000 / 60 / 60 / 24)
+      : 0;
     const checked = event.target.checked;
     if (checked) {
       const itemRef = doc(db, 'shopping-list', id);
-      setDoc(itemRef, { lastPurchasedDate: date.getTime() }, { merge: true });
+      setDoc(
+        itemRef,
+        {
+          lastPurchasedDate: date.getTime(),
+          estimatedPurchaseDate: calculateEstimate(
+            item.estimatedPurchaseDate,
```

Comment:
I'm not sure, but I just have asked  in the slack, but I think, here you need to send the values in the attribute `days`  that select in the radio button

Reply:
Thank you, we've incorporated this suggestion!